src/components/deviceprofileBuilder.ts

```diff
@@ -267,32 +278,48 @@ function getTranscodingProfiles(): Array<TranscodingProfile> {
     const TranscodingProfiles: Array<TranscodingProfile> = [];
 
     const hlsAudioCodecs = getSupportedHLSAudioCodecs();
-    const audioChannels: number = hasSurroundSupport(currentDeviceId) ? 6 : 2;
 
     if (profileOptions.enableHls !== false) {
+        // HLS for audio only
         TranscodingProfiles.push({
             Container: 'ts',
             Type: DlnaProfileType.Audio,
             AudioCodec: hlsAudioCodecs.join(','),
             Context: EncodingContext.Streaming,
             Protocol: 'hls',
-            MaxAudioChannels: audioChannels.toString(),
+            // only 2ch supported for audio
+            MaxAudioChannels: '2',
             MinSegments: 1,
             BreakOnNonKeyFrames: false
         });
     }
 
     const supportedAudio = getSupportedAudioCodecs();
+    const surroundCodecs = getSupportedSurroundCodecs();
 
     // audio only profiles here
     for (const audioFormat of supportedAudio) {
+        // direct download audio
         TranscodingProfiles.push({
             Container: audioFormat,
             Type: DlnaProfileType.Audio,
             AudioCodec: audioFormat,
             Context: EncodingContext.Streaming,
             Protocol: 'http',
-            MaxAudioChannels: audioChannels.toString()
+            // only 2ch supported for audio
+            MaxAudioChannels: '2'
+        });
+    }
+
+    // Passthrough for audio only
+    if (surroundCodecs.length) {
+        // surround audio, try mkv here
+        TranscodingProfiles.push({
+            Container: 'mkv',
+            Type: DlnaProfileType.Audio,
+            AudioCodec: surroundCodecs.join(','),
+            Context: EncodingContext.Streaming,
+            Protocol: 'http'
```

Comment:
I have no idea if this will work, but I'm trying to tell jellyfin to send me an ac3 audio file muxed in mkv

Reply:
Audio doesn't do passthrough audio